src/ui/index.js

```diff
@@ -1,24 +1,26 @@
 import Vue from 'vue'
-import vuetify from './plugins/vuetify'
```

Comment:
I am sorry for these import changes. My vscode has always been auto-sorting on save, and I've had to compromise after trying various things.